addons/missileguidance/stringtable.xml

```diff
@@ -87,6 +93,7 @@
             <English>PRE-PLANNED</English>
             <French>PRE-PLANIFICATION</French>
             <Italian>PRE-PLANNED</Italian>
+            <German>PRE-PLANNED</German>
```

Comment:
Vorprogrammiert vielleicht?

Reply:
Dann müsste man auch VP oder so für die kurze Schreibweise verwenden. Und TARGET OF OPPORTUNITY müsste man auch anpassen. Ich weiß nicht, ob es dafür gute Übersetzungen gibt. In der Luftfahrt ist halt alles Englisch.

Reply:
Okay, passt.